apps/web/src/app/domain/pages/blocks/blocks.page.html

```
@@ -9,49 +9,53 @@
-        loading="lazy"
-        [src]="getCurrentBlockImage(block)"
-        [alt]="block.title"
-        class="border-border h-[220px] w-full rounded-lg border shadow-sm"
-      />
+    @if (blocks().length > 0) {
+      @for (block of blocks(); track block.id) {
+        <div class="space-y-3 lg:hidden">
+          <h3 class="text-foreground text-sm font-semibold">{{ block.title }}</h3>
+          @if (block.image) {
+            <img
+              width="1440"
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)